core/src/main/scala/chisel3/Module.scala

```diff
@@ -85,6 +86,8 @@ object Module extends SourceInfoDoc {
   def reset: Reset = Builder.forcedReset
   /** Returns the current Module */
   def currentModule: Option[BaseModule] = Builder.currentModule
+  /** Returns the global Annotations */
+  def annotationSeq: AnnotationSeq = Builder.annotationSeq
```

Comment:
```suggestion
  /** Returns the global Annotations */
  def annotations: AnnotationSeq = Builder.annotationSeq
```

I'm also not sure if `Module` is the right place for this, but I'm not really sure where else would be right either...

Reply:
When I was working on this, I also get puzzled where to put this API.
Maybe directly under `chisel3.experimental`?
But I'm not sure is experimental thread safe for this API?

Reply:
Since these annotations are set at the beginning of elaboration (ie. they aren't modified), I don't think thread-safety is really a concern fortunately